lab1/Syntax.in.txt

```diff
@@ -0,0 +1,23 @@
+program = "begin" ";" decllist ";" cmpdstmt end ";"
```

Comment:
Here you don't need ";" after decllist, because you already have a ";" after each declaration. Also be careful with cmpdstmt: it contains "{}", while in your code the statements don't start with curly brackets.